test/test.mjs

```diff
@@ -1094,7 +1100,7 @@ function main() {
       startUnitTest("/test/unit/unit_test.html", "unit");
     });
   } else if (options.fontTest) {
-    startUnitTest("/test/font/font_test.html", "font");
+    startUnitTest("/test/font/font_test.html", "font", /* headless = */ true);
```

Comment:
To aid with *local* testing, can we perhaps limit headless testing to only GitHub Actions?
Perhaps we could utilize the existing `masterMode` test-flag, somehow, to conditionally enable headless testing?

Reply:
Yes, I'll have a look at that. I must admit I have been going back and forth on this, and in the end went for this approach because it seemed simplest so we don't have to special-case local runs versus GitHub Actions runs, and I personally didn't really find the browser windows all that useful locally. However, I can see why it would be preferred and as said I was in doubt about this change, so I'll change this (most likely the only way to do this is to be headful by default and create a `gulp fonttest --headless` flag for GitHub Actions runs so they can opt-in to headless mode and the local experience doesn't change.

Reply:
Actually, something that just occurred to me, does `gulp fonttest` still even work *locally* with this patch?
Given that all Python dependencies are being removed from the repository, can this even run locally now unless you *manually* install the necessary packages?

Reply:
Yes, this still works locally, but it does indeed require installing `fonttools`. I have put instructions in the commit message of https://github.com/mozilla/pdf.js/pull/17263/commits/4f4065432157df34ee4a635c0b7e4feaaeb37750 on how to do that, but it doesn't necessary require a virtual environment: any installation of `fonttools` that puts `ttx` on the path will work, so also global installations.

Note that also before a manual installation of Python and a submodule clone was necessary (both were not performed by default), so in that sense not much changes: the font tests were never runnable out of the box without installing the dependencies manually.

If it helps, I can write up the instructions for running the font tests locally in a README file in the tests folder (similar to the README file we had there before) in which I list exactly which dependencies are required, where to fetch them from and how to install them (basically moving what I wrote in the commit message to an in-repo README and extending it)?

Reply:
*Sorry, should've read the commit message more closely.*

Could you perhaps include the instructions, regarding setup for local testing, in an error message e.g. in the `ttxdriver.mjs` file to ensure that this is more clearly visible?

Reply:
> Would it help if I write up the instructions for running the font tests locally in a README file in the tests folder (similar to the README file we had there before)? I can then list which dependencies are needed, where to get them from and how to install them (basically moving what I wrote in the commit message to an in-repo README and extending it).

Yes, please do that! That'll really help ensure that this information is available, and it'll be easy to update if/when necessary.